src/plugins/data/public/query/filter_manager/filter_manager.ts

```diff
@@ -45,20 +45,23 @@ export class FilterManager {
     const appFilters = partitionedFilters.appFilters;
 
     // existing globalFilters should be mutated by appFilters
+    // ignore original appFilters which are already inside globalFilters
+    const cleanedAppFilters: esFilters.Filter[] = [];
```

Comment:
I have this found bug fix as separate pr - 
https://github.com/elastic/kibana/pull/55591